template/app/src/main/res/values/styles-text.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="TextAppearance.Bloco.ToolbarTitle" parent="TextAppearance.MaterialComponents.Headline1">
```

Comment:
Here usually goes the name of the app (`Template`) instead of the name of the company (`Bloco`). The same for TextAppearances. But that's just a small detail. It is something that the start script would also need to worry about.